theRoughCode/WatsMyMajor
# Remove avatar button

# Description

Added a button to the settings page Change Avatar dialog to remove the current avatar image
Added @material-ui/core to the depenencies.

Fixes #60 

## Type of change

# How Has This Been Tested?
- Log in with any user and navigate to Settings > Change Avatar
- Press Upload Image and submit to change the avatar picture
- Press Change Avatar again and press Remove Avatar
- The Avatar should be the default Goose picture
- Press Change Avatar once more, and the Remove Avatar button should be disabled

# Checklist:
